docs/configuration.txt

```diff
@@ -1567,6 +1572,48 @@ And here is a snippet of the generated properties ready for use with log4j. You
 
    ...
 
+Logger Settings
+---------------
+
+It's possible to set the log level of loggers at runtime. This is particularly
+useful when debugging problems and there is a need to increase the logging
+setting without wanting to restart nodes. Logging settings are cluster wide and
+override the logging configuration of nodes defined in their ``logging.yaml``.
+
+The :ref:RESET <ref-set> statement is not supported for logging settings.
+Therefore, it is recommended to change log levels only using TRANSIENT
+because they will be discarded on cluster restart and the logging configuration
+defined in logging.yaml will be applied.
+If using PERSISTENT it would not possible to discard them to revert to the
+values defined in the logging configuration.
+
+To set the log level you can use the regular :ref:`SET<ref-set>` statement,
+for example:
+
+.. code-block:: sql
+
+  SET GLOBAL TRANSIENT "logger.action" = 'INFO';
+
+The logging setting is a wildcard setting consisting of the prefix ``logger``
```

Comment:
Would omit the `is a wildcard setting`:

The logging setting consists of the prefix `logger` and a suffix...